src/attrcache.c

```diff
@@ -234,7 +234,6 @@ int git_attr_cache__get(
 		/* remove existing entry */
 		if (file) {
 			attr_cache_remove(cache, file);
-			git_attr_file__free(file); /* offset incref from lookup */
```

Comment:
Yeah, I updated this - when `attr_cache_lookup_entry` fails to find the entry, it frees the given entry.  (Which would lead to the double-free that coverity complains about.)